server/src/auth.ts

```diff
@@ -5,19 +5,20 @@ import { Strategy as JwtStrategy, ExtractJwt } from 'passport-jwt';
 import { scryptSync } from 'crypto';
 import { dbClient, collections } from './db';
 import logger from './logger';
+import { User } from './types';
```

Comment:
I suggest we use types folder and create a file for each type

Reply:
Got it. I'll refactor types.